bravado/client.py

```diff
@@ -271,15 +282,16 @@ def response_callback(incoming_response, operation):
     raise_on_unexpected(incoming_response)
 
     try:
-        swagger_return_value = unmarshal_response(incoming_response, operation)
+        incoming_response.swagger_result = unmarshal_response(
```

Comment:
i think we should then document that the result interface of `incoming_response` is not exactly the one defined in `bravado_core.response.IncomingResponse`, but has some extra attributes added.